src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -466,6 +489,11 @@ private void updateBundleAndRecipientListWithResponsesForStudent(
 
         bundle.put(question, responses);
         recipientList.put(question.getId(), recipients);
+        for (FeedbackResponseAttributes response : responses) {
```

Comment:
The method is no responsible to fetch comment.

Define a new method to do the job.